theme/css/general.css

```diff
@@ -180,3 +180,9 @@ blockquote {
 .result-no-output {
     font-style: italic;
 }
+
+.nav-return,
+.nav-return:visited {
+    color: var(--searchbar-fg);
```

Comment:
The color should be --sidebar-fg to reflect the color of the text for all the different themes (light,ayu etc). --searchbar-fg is always black

Reply:
thanks, I'll push a fix now